packages/app/obojobo-document-engine/src/scripts/viewer/stores/question-store.js

```diff
@@ -71,8 +71,7 @@ class QuestionStore extends Store {
 
 			'question:showExplanation': payload => {
 				const { id, context } = payload.value
-				const root = OboModel.models[id].getRoot()
-
+				const root = OboModel.getRoot()
```

Comment:
I need input from @zachberry about using `OboModel.getRoot().get('draftId')`  It's working to solve the problem here, but it seems a little sketchy.

The problem - When used in assessment review, the models for the review questions are not part of a complete document tree.  They cannot follow the `parent.parent.parent` chain to the top to get the draftId (what OboModel attempts to do). 

I suspect my fix here works by accident - talking to OboModel.getRoot() directly just so happens to reference the first OboModel that was created - which happens to be the one for the document - which happens to have access to draftId.

I'd like to either create a global draftId in the viewer/editor or more intentionally propagate that to OboModel, instead of by accident.

Reply:
Agree, the OboModel.getRoot().get('draftId') always felt lame to me but it was working so we ran with it. I'd much rather have a way to get a global draftId.